### ISSUE TITLE
Should `webpack.plugins` nesting in the configuration file go away?

### ISSUE THREAD

**Issue by insin** [insin/nwb#113]
Just realised `webpack.plugins` is really an implementation detail - it's just a handy way to group certain configuration (`define`, `html` etc.) which will be applied to the default set of Webpack plugins managed by nwb in a particular function, but that's not really relevant to the end user.

Should these all be moved up to `webpack`?

Before:
```js
  webpack: {
    loaders: {
      css: {
        query: {
          modules: true,
          localIdentName: '[hash:base64:5]'
        }
      }
    },
    plugins: {
      define: {
        __GAPI_KEY__: JSON.stringify('YOURAPIKEY')
      },
      html: {
        mountId: 'root',
        title: 'The Cinnamons'
      }
    },
    postcss: {
      plugins: [
        require('precss'),
        require('autoprefixer'),
        require('cssnano')
      ]
    }
  }
```

After:

```js
  webpack: {
    define: {
      __GAPI_KEY__: JSON.stringify('YOURAPIKEY')
    },
    html: {
      mountId: 'root',
      title: 'The Cinnamons'
    },
    loaders: {
      css: {
        query: {
          modules: true,
          localIdentName: '[hash:base64:5]'
        }
      }
    },
    postcss: {
      plugins: [
        require('precss'),
        require('autoprefixer'),
        require('cssnano')
      ]
    }
  }
```

**Comment by insin** [comment]
Other potential simplifications:

* Don't require the `query` key for loader config tweaks, as it's the most common thing to change - special-case `include` and `exclude` if they're provided (doe this clash with any load query settings?)
* Apply `postcss` config to the default style pipeline if an Array is given.

```js
  webpack: {
    define: {
      __GAPI_KEY__: JSON.stringify('YOURAPIKEY')
    },
    html: {
      mountId: 'root',
      title: 'The Cinnamons'
    },
    loaders: {
      css: {
        modules: true,
        localIdentName: '[hash:base64:5]'
      }
    },
    postcss: [
      require('precss'),
      require('autoprefixer'),
      require('cssnano')
    ]
  }
```

**Comment by insin** [comment]
All implemented in a25ac149b24a59be1ffb8708d423623e31d9812f and aaf7ef7c2ec90ac325f1fde57e88f452272f307a
